drs/drs-assembly/subscribe-model.conf

```diff
@@ -10,246 +10,86 @@ Items with the FITS= tag in the description refer to general meta-data from
-  usage = "Guidestar 2 Dec. FITS=GUIDDEC2"
-}
-
-{
-  subsystem =  AOESW
-  component =  aoesw-tbd
-  name =  GUIDDEC3
-  requiredRate =  4
-  usage = "Guidestar 3 Dec. FITS=GUIDDEC3"
-}
-
-{
-  subsystem =  AOESW
-  component =  aoesw-tbd
-  name =  GUIDNAM1
-  requiredRate =  4
-  usage = "Guidestar 1 name. FITS=GUIDNAM1"
-}
-
-{
-  subsystem =  AOESW
-  component =  aoesw-tbd
-  name =  GUIDNAM2
-  requiredRate =  4
-  usage = "Guidestar 2 name. FITS=GUIDNAM2"
-}
-
-{
-  subsystem =  AOESW
-  component =  aoesw-tbd
-  name =  GUIDNAM3
-  requiredRate =  4
-  usage = "Guidestar 3 name. FITS=GUIDNAM3"
-}
-
-{
-  subsystem =  AOESW
-  component =  aoesw-tbd
-  name =  GUIDPM1
-  requiredRate =  4
-  usage = "Guidestar 1 proper motion. FITS=GUIDPM1"
-}
-
-{
-  subsystem =  AOESW
-  component =  aoesw-tbd
-  name =  GUIDPM2
-  requiredRate =  4
-  usage = "Guidestar 2 proper motion. FITS=GUIDPM2"
-}
-
-{
-  subsystem =  AOESW
-  component =  aoesw-tbd
-  name =  GUIDPM3
-  requiredRate =  4
-  usage = "Guidestar 3 proper motion. FITS=GUIDPM3"
-}
+  events = [
 
 {
-  subsystem =  AOESW
-  component =  aoesw-tbd
-  name =  GUIDRA1
+  subsystem =  TCS
+  component =  cmIRIS
+  name =  oiwfsTargetInfo
   requiredRate =  4
-  usage = "Guidestar 1 RA. FITS=GUIDRA1"
+  usage = "Proposed item in which TCS publishes ICRS RA, Dec and proper motions, names, and colors (what system?) for OIWFS targets. FITS=GUIDDEC*, GUIDRA*, GUIDNAM*, GUIDPM*, GUIDCOL*"
 }
 
 {
-  subsystem =  AOESW
-  component =  aoesw-tbd
-  name =  GUIDRA2
+  subsystem =  TCS
+  component =  cmIRIS
+  name =  odgwTargetInfo
   requiredRate =  4
-  usage = "Guidestar 2 RA. FITS=GUIDRA2"
+  usage = "Proposed item in which TCS publishes ICRS RA, Dec and proper motions, names, and colors (what system?) for ODGW targets. FITS=???"
 }
 
 {
-  subsystem =  AOESW
-  component =  aoesw-tbd
-  name =  GUIDRA3
+  subsystem =  TCS
+  component =  cmNFIRAOS
+  name =  ngsTargetInfo
   requiredRate =  4
-  usage = "Guidestar 3 RA. FITS=GUIDRA3"
+  usage = "Proposed item in which TCS publishes ICRS RA, Dec and proper motions, names, and colors (what system?) for the NGS WFS target. FITS=???"
 }
 
 {
-  subsystem =  AOESW
-  component =  aoesw-tbd
-  name =  GUIDWAV1
+  subsystem =  TCS
+  component =  cmIRIS
+  name = oiwfs1AtmDispersion
   requiredRate =  4
-  usage = "Guidestar 1 wavelength (microns). FITS=GUIDWAV1"
+  usage = "Guidestar 1 wavelength (microns). Derive from referenceWavelength or weight? FITS=GUIDWAV1"
 }
 
 {
-  subsystem =  AOESW
-  component =  aoesw-tbd
-  name =  GUIDWAV2
+  subsystem =  TCS
+  component =  cmIRIS
+  name = oiwfs2AtmDispersion
   requiredRate =  4
   usage = "Guidestar 2 wavelength (microns). FITS=GUIDWAV2"
 }
 
 {
-  subsystem =  AOESW
-  component =  aoesw-tbd
-  name =  GUIDWAV3
+  subsystem =  TCS
+  component =  cmIRIS
+  name = oiwfs3AtmDispersion
   requiredRate =  4
   usage = "Guidestar 3 wavelength (microns). FITS=GUIDWAV3"
 }
 
 {
-  subsystem =  IRIS
```

Comment:
I think that this next block of AO keywords was perhaps too simplistic. Need to discuss how NFIRAOS tier assignments work, and some of the main loop rates (e.g., HO and LO, truth etc.) with an AO person to figure out a sensible subset.